### PR TITLE
[READY] BYOND 516 compatability

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
Fixes TGUI to work with webview2 and associated 516 changes, continuation of (and supersedes) https://github.com/ParadiseSS13/Paradise/pull/25021

Internal BYOND href formats have been updated and will be enforced by CI, instead of `href='?` we now require `href='byond://?`

This can go on TM once all 515 regressions are fixed, aim is to allow bridging between both versions for upcoming beta cycle.

### TODO:

- [x] Figure out why TGchat crashes on 515 every other load (probably something to do with saving/loading history)
- [x] Figure out why statbrowser stopped receiving updates on 515 when I didn't touch it (works fine on 516?????)
### Issues on 516 clients can be addressed later as the alpha/beta refines
- [ ] ~~Make 516 statbrowser properly inherit theme on initial load~~
- [ ] ~~Ensure 516 TGchat settings save properly~~
- [ ] ~~Ensure TGUIsay steals focus for first load on 516~~
- [ ] ~~Prevent F5 refresh for chat and stat panes on 516~~

## Why It's Good For The Game
Working browsers

## Testing
By 516 alpha build users, don't worry about it for now.

## Changelog
:cl: AffectedArc07, S34NW
fix: Made TGUI work on 516
Add: New maintainer-only verb that lets you use inspect element on live TGUI
/:cl:

### CODE REVIEW

**Comment by 1080pCat** [comment]
first bug of TM :)
image
tg chat gets a weird scroll icon when it really shouldn't

**Comment by AffectedArc07** [comment]
```
[warn] packages/tgui-say/styles/button.scss
[warn] packages/tgui-say/styles/content.scss
[warn] packages/tgui-say/styles/dragzone.scss
[warn] packages/tgui-say/styles/textarea.scss
[warn] packages/tgui-say/styles/window.scss
[warn] Code style issues found in 5 files. Run Prettier to fix.
```

**Comment by S34NW** [comment]
Prettier not running for some reason, it's a style thing only so should be fine

**Comment by FunnyMan3595** [comment]
A robotics brain has been activated outside the box:
image

**Comment by AffectedArc07** [comment]
> A robotics brain has been activated outside the box: image

Artefact of when robotocists put them down on the table and the object gets pixel_x and pixel_y variables. They escape the bounding box and have done since we added these popups.

**Comment by S34NW** [comment]
> A robotics brain has been activated outside the box: image

Please make an issue report instead of commenting on a random PR

**Review by AffectedArc07 — APPROVED** [review]
Approved what I own

**Comment by Contrabang** [comment]
I think this may currently break the job assets that are included as antaghud as a part of the orbitting menu? As introduced in #24578 ?

**Comment by S34NW** [comment]
> I think this may currently break the job assets that are included as antaghud as a part of the orbitting menu? As introduced in #24578 ?

That is unusual, I didn't touch anything relating to that but it may be a possibility

File: code/modules/admin/permissionverbs/permissionedit.dm [hunk 1]
```diff
@@ -23,8 +23,8 @@
 <body onload='selectTextField();updateSearch();'>
 <div id='main'><table id='searchable' cellspacing='0'>
 <tr class='title'>
-<th style='width:125px;text-align:right;'>CKEY <a class='small' href='?src=[UID()];editrights=add'>\[+\]</a></th>
-<th style='width:125px;'>RANK</th><th style='width:100%;'>PERMISSIONS</th>
+<th style='width:20%;text-align:right;'>CKEY <a class='small' href='byond://?src=[UID()];editrights=add'>\[+\]</a></th>
+<th style='width:20%;'>RANK</th><th style='width:80%;'>PERMISSIONS</th>
```
Comment (S34NW) [inline code/modules/admin/permissionverbs/permissionedit.dm:27]:
```suggestion
<th style='width:20%;'>RANK</th><th style='width:60%;'>PERMISSIONS</th>
```

**Review by Burzah — APPROVED** [review]

**Review by Contrabang — APPROVED** [review]
FISI

**Review by AffectedArc07 — APPROVED** [review]
# FULL SEND

**Pull request merged**